pkg/scripts/testdata/TestUpgradeKubeadmAndCNIAmazonLinux.golden

```diff
@@ -74,7 +74,7 @@ sudo yum versionlock delete docker containerd || true
 
 sudo yum install -y \
 	docker-'19.03.*' \
-	containerd.io-'1.4.*'
+	containerd.io-'1.5.*'
```

Comment:
Hmm, I think this should not happen

Reply:
I found the issue, I'll try to push the fix later.

Reply:
@kron4eg Fixed, PTAL.